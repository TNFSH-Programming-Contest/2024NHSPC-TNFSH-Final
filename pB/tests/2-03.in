22 12
3 2 10 2 5 1 4 8 7 9 6 2 6 9 7 6 8 1 1 9 6 6
205639 146 311294069 3073 115 302983 609 59195 295 71517693 750720 1375666
0 
4 12 7 5 4
95
0 
7 2 11 4 6 1 8 10
8
0 
10 5 7 10 2 12 11 8 4 9 6
97
0 
8 2 3 6 7 5 8 12 10
40
0 
10 6 7 10 9 11 3 2 12 4 5
93
0 
8 7 6 8 5 11 1 9 4
26
0 
0 
55
0 
3 3 9 7
18
0 
7 9 11 10 1 5 6 12
84
0 
6 12 4 3 11 5 8
49
0 
0 
91
0 
0 
84
0 
1 3
48
0 
8 10 1 5 11 4 3 2 9
7
0 
1 4
8
0 
7 3 9 2 1 12 5 6
100
0 
2 10 4
31
0 
3 2 10 4
10
0 
2 2 9
24
0 
7 12 3 5 1 4 10 11
47
0 
8 10 3 6 11 9 5 7 8
61
0 
5 5 4 6 1 9
78
0 
8 12 4 10 5 3 9 6 7
100
0 
9 7 9 2 3 6 10 4 8 11
36
0 
1 12
53
0 
1 10
57
0 
5 10 2 9 8 12
74
0 
5 11 12 7 6 10
65
0 
2 9 8
22
0 
4 11 2 5 1
4
0 
9 8 3 9 1 5 2 4 7 6
36
0 
6 8 12 5 11 2 10
15
0 
0 
78
0 
2 7 8
6
0 
10 3 2 6 4 1 8 5 11 7 12
92
0 
9 1 10 12 7 5 3 6 4 8
97
0 
1 7
33
0 
3 5 6 1
41
0 
0 
99
0 
7 12 4 3 2 10 9 1
99
0 
4 11 6 9 2
1
0 
6 4 7 10 6 1 8
30
0 
7 4 12 10 7 5 11 8
15
0 
0 
28
0 
10 6 5 2 3 4 9 8 1 11 12
60
0 
2 6 5
92
0 
3 12 2 5
21
0 
9 12 7 9 10 8 6 1 5 4
40
0 
0 
85
0 
7 3 6 2 10 9 7 5
31
0 
2 10 6
53
0 
0 
39
0 
9 5 10 1 7 2 12 9 6 4
1
0 
4 5 2 9 10
52
0 
9 4 2 11 5 3 7 9 1 10
10
0 
6 10 2 8 6 9 5
17
0 
2 11 12
70
0 
7 3 2 7 10 5 4 6
100
0 
8 3 1 12 11 7 2 10 8
25
0 
3 2 3 6
25
0 
7 6 11 3 9 2 7 12
35
0 
7 1 8 9 12 2 6 11
32
0 
7 11 3 4 9 6 1 7
52
0 
0 
24
0 
6 12 11 10 7 6 5
90
0 
2 4 6
21
0 
7 8 9 2 12 1 6 4
84
0 
1 8
90
0 
7 8 6 2 4 5 11 10
74
0 
9 6 3 4 10 7 8 5 12 1
20
0 
7 10 11 4 9 7 1 8
45
0 
4 3 6 4 7
87
0 
3 3 7 12
99
0 
0 
9
0 
4 3 9 12 1
31
0 
2 12 7
73
0 
5 4 3 12 8 2
13
0 
8 2 12 6 3 9 4 10 1
54
0 
2 8 11
45
0 
9 2 3 4 6 9 5 8 1 7
40
0 
5 8 7 9 3 4
21
0 
5 7 3 9 5 12
34
0 
5 8 10 2 7 12
21
0 
2 10 7
98
0 
10 3 5 12 2 4 1 8 7 6 11
45
0 
3 11 6 1
54
0 
6 9 11 2 6 12 7
28
0 
3 5 1 6
4
0 
4 4 3 5 9
5
0 
7 9 10 3 6 5 4 12
38
0 
7 2 4 10 1 6 7 3
88
0 
5 11 3 7 5 1
4
0 
8 10 12 11 1 9 5 8 4
24
0 
0 
54
0 
3 3 6 11
11
0 
5 6 2 8 4 10
12
0 
4 11 4 3 12
82
0 
10 1 3 7 10 5 9 4 2 8 6
38
0 
7 7 9 11 3 1 10 5
51
0 
0 
25
0 
6 12 3 9 1 8 4
95
0 
7 9 5 8 10 11 1 4
23
0 
10 7 3 1 6 5 2 12 9 8 11
14
0 
9 11 7 1 4 6 12 10 9 2
29
0 
4 10 8 5 6
31
0 
8 8 9 10 3 1 11 4 12
31
0 
0 
73
0 
1 7
64
0 
6 5 12 8 10 11 9
47
0 
4 5 2 3 11
91
0 
9 12 11 6 3 8 1 2 4 10
14
0 
2 1 10
82
0 
9 8 7 12 2 3 4 5 10 11
18
0 
10 6 8 5 3 10 1 12 9 2 4
65
0 
3 9 5 12
95
0 
8 1 5 3 7 4 11 12 6
9
0 
4 8 4 10 1
24
0 
7 7 4 2 1 12 11 5
21
