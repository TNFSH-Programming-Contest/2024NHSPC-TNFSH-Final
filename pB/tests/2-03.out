1379558
384550516
55
71520797
465
384494603
1375719
78
99
28
39
383118788
24
9
809960
1259396
54
71883102
313423610
25
73
1376171
